nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,92 @@
+/*Calculator
+Let's again create a calculator. This time the calculator will support 4 features:
+
+Addition
+Subtraction
+Multiplication
+Division*/
+
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({
+    extended: true
+}));
+
+// welcome screen
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+//Getting data through query parameters using GET
+
+app.get("/calculator/add", (req, res) => {
+    try {
+	const queryParam = Object.values(req.query);
+	const data = queryParam.map(Number);
+	const addition = data.reduce((result, value) => (result = result + value));
+	isNaN(addition) ? res.status(400).send("Invalid input") : res.send(`Query Params are: ${queryParam} Addition is equal to: ${addition}`);
+    } catch (error) {
+        response.status(404).send("values undefined");
+    }
+});
+
+app.get("/calculator/multiply", (req, res) => {
+    try{
+	const queryParam = Object.values(req.query);
+	const data = queryParam.map(Number);
+	const multiplication = data.reduce((result, value) => (result = result * value), 1);
+	isNaN(multiplication) ? res.status(400).send("Invalid input") : res.send(`Query Params are: ${queryParam} Multiplication is equal to: ${multiplication}`);
```

Comment:
This route, and also the other ones, should be able to handle repeated parameters like this:

```
/calculator/multiply?firstParam=1&secondParam=2&secondParam=4
```

But if I try that with your implementation, I get `Invalid input`.